### PR TITLE
LPS-65469

### PR DESCRIPTION
**Pull request by Ithildir**

### CODE REVIEW

File: modules/util/portal-tools-db-upgrade-client/src/main/java/com/liferay/portal/tools/db/upgrade/client/UpgradeClient.java [hunk 1]
```diff
@@ -375,10 +340,37 @@ private String _getRelativeFileName(Path basePath, Path path) {
...
+
+			return true;
+		}
+	}
+
 	private void _printHelp() {
 		System.out.println("\nUpgrade commands:");
 		System.out.println("exit or quit - exit Gogo Shell");
 		System.out.println("upgrade:help - show upgrade commands");
+		System.out.println("upgrade:dryRun - List the upgrades that failed");
```
Comment (Ithildir) [inline modules/util/portal-tools-db-upgrade-client/src/main/java/com/liferay/portal/tools/db/upgrade/client/UpgradeClient.java]: @david-truong, we can't sort this line, right?

**Comment by brianchandotcom** [comment]
ci:stop

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/d656a67e85...dbef150587

**Comment by brianchandotcom** [comment]
@david-truong 

You should also clean up

```
	private void _printHelp() {
		System.out.println("\nUpgrade commands:");
		System.out.println("exit or quit - Exit Gogo Shell");
		System.out.println("upgrade:dryRun - List unfinished upgrades");
		System.out.println(
			"upgrade:execute {module_name} - Execute upgrade for specified " +
				"module");
		System.out.println("upgrade:help - Show upgrade commands");
		System.out.println("upgrade:list - List registered upgrades");
		System.out.println(
			"upgrade:list {module_name} - List upgrade steps required for " +
				"specified module");
		System.out.println(
			"upgrade:list | grep Registered - List registered upgrades and " +
				"their current version");
		System.out.println(
			"upgrade:list | grep Registered | grep steps - List upgrades in " +
				"progress");
		System.out.println(
			"verify:execute {module_name} - Execute verifier for specified " +
				"module");
		System.out.println("verify:list - List registered verifiers");
	}
```

Does "dryRun" return failed or unfinished upgrades? I see inconsistent var names. See dbef150587723880e862b011d851aa034c0fec2e

Please send a fix @david-truong thx.

File: modules/util/portal-tools-db-upgrade-client/src/main/java/com/liferay/portal/tools/db/upgrade/client/UpgradeClient.java (hunk 1 above)
Comment (david-truong) [inline modules/util/portal-tools-db-upgrade-client/src/main/java/com/liferay/portal/tools/db/upgrade/client/UpgradeClient.java]: I'm not opposed to it.  I've was going back on forth on whether I should.

**Comment by david-truong** [comment]
@brianchandotcom I talked to the upgrade team and we are in the process of renaming that command as the name is very confusing.  Once it gets accepted I will make the change here.  It will list upgrades that did not get upgraded completely for whatever reason (stopped/failed)

**Pull request closed without merging**